server/lib/amplitude.js

```diff
@@ -60,7 +48,7 @@ const VIEW_EVENT_GROUPS = {
 };
 
 const CONNECT_DEVICE_FLOWS = {
-  'connect-another-device': 'cad',
```

Comment:
Another happy side-effect is that using the `install_from` and `signin_from` events to trigger the view event means we know whether to set `connect_device_flow` to `store_buttons` or `signin` and can ditch this `cad` value (which was only added as a hack).